schemas/error-declaration-list.yaml

```diff
@@ -21,10 +21,6 @@ $defs:
       description:
         type: string
         minLength: 2
-      namespace:
-        type: string
-        minLength: 2
-        pattern: ^[a-z][a-z0-9]*$
```

Comment:
Not really related to supporting errors in EVerest, I think this should be a part of a different MR

Reply:
I've discussed this with @andistorm - this just updates the schema